packages/nuxt-locokit/src/paths.ts

```diff
@@ -24,26 +33,30 @@ export const ROUTES_PATH = {
    * Auth paths
    */
   AUTH: {
-    LOSTPASSWORD: '/auth/lost-password',
-    VERIFYSIGNUP: '/auth/verify-signup',
-    RESETPASSWORD: '/auth/reset-password',
-    SIGNUP: '/auth/signup',
-    SIGNIN: '/auth/signin',
+    LOST_PASSWORD: '/auth/lost-password',
+    VERIFY_SIGNUP: '/auth/verify-signup',
+    RESET_PASSWORD: '/auth/reset-password',
+    SIGN_UP: '/auth/sign-up',
+    SIGN_IN: '/auth/sign-in',
+    ALREADY_AUTHENTICATED: '/auth/already-authenticated',
   },
 
   /**
    * User paths
    */
-  USER: {
-    PROFILE: '/user/profile',
-    UPDATEEMAIL: '/user/update-email',
+  PROFILE: {
+    HOME: '/profile',
+    UPDATE_GENERAL: '/profile/general',
+    UPDATE_EMAIL: '/profile/update-email',
+    UPDATE_PASSWORD: '/profile/update-password',
```

Comment:
To be consistent, either we add the `update-` prefix to all routes, neither we don't.
I prefer a cleaner URL, so without it's ok for me.

```suggestion
    UPDATE_GENERAL: '/profile/general',
    UPDATE_EMAIL: '/profile/email',
    UPDATE_PASSWORD: '/profile/password',
```